crates/aptos-batch-encryption/Cargo.toml

```diff
@@ -30,15 +30,15 @@ ark-poly = { workspace = true }
 ark-serialize = { workspace = true }
 ark-std = { workspace = true }
 bcs = { workspace = true }
-ed25519-dalek = { version = "2.1.1", features = ["asm", "rand_core", "serde"] }
+ed25519-dalek = { version = "2.1.1", features = ["serde"] }
 generic-array = { workspace = true }
 hmac = { workspace = true }
 num-traits = { workspace = true }
 rand = { workspace = true }
 rayon = { workspace = true }
 serde = { workspace = true }
 serde_json = { workspace = true }
-sha2 = { version = "0.10.8", features = ["asm", "asm-aarch64", "sha2-asm"] }
+sha2 = { version = "0.10.6" }
```

Comment:
what happens if you set this to `workspace = true`?

Reply:
it resolves to sha2 0.9.3 and i got compile error